### PR TITLE
fix: Fix translation of application status. feat: Add BorrowerSector. Use gettext to translate enums.

### PR DESCRIPTION
**Pull request by jpmckinney**
#362

### CODE REVIEW

**Comment by jpmckinney** [comment]
@yolile FYI this PR contains a fix for #357, where the capitalized application statuses in `applications/export/{lang}` wouldn't be translated. You can merge it already if needed. Otherwise I'll add more commits for #362

**Comment by yolile** [comment]
Feel free to add more commits

**Comment by jpmckinney** [comment]
@yolile There are some error messages for you to translate (Or, we can let them continue to be returned in English, and relax the i18n.yml workflow). You can add your commits to finish #362 (or create a new PR).

**Comment by yolile** [comment]
I get this error when I run` pybabel extract -F babel.cfg -o messages.pot .`

```
extracting messages from README.rst (classes="ApplicationStatus,BorrowerDocumentType,BorrowerSize,BorrowerSector")
Traceback (most recent call last):
  File "/home/yolile/development/credere-backend/.ve/bin/pybabel", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/yolile/development/credere-backend/.ve/lib/python3.11/site-packages/babel/messages/frontend.py", line 999, in main
    return CommandLineInterface().run(sys.argv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yolile/development/credere-backend/.ve/lib/python3.11/site-packages/babel/messages/frontend.py", line 925, in run
    return cmdinst.run()
           ^^^^^^^^^^^^^
  File "/home/yolile/development/credere-backend/.ve/lib/python3.11/site-packages/babel/messages/frontend.py", line 523, in run
    for filename, lineno, message, comments, context in extracted:
  File "/home/yolile/development/credere-backend/.ve/lib/python3.11/site-packages/babel/messages/extract.py", line 217, in extract_from_dir
    yield from check_and_call_extract_file(
  File "/home/yolile/development/credere-backend/.ve/lib/python3.11/site-packages/babel/messages/extract.py", line 281, in check_and_call_extract_file
    for message_tuple in extract_from_file(
                         ^^^^^^^^^^^^^^^^^^
  File "/home/yolile/development/credere-backend/.ve/lib/python3.11/site-packages/babel/messages/extract.py", line 323, in extract_from_file
    return list(extract(method, fileobj, keywords, comment_tags,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/yolile/development/credere-backend/.ve/lib/python3.11/site-packages/babel/messages/extract.py", line 441, in extract
    raise ValueError(f"Unknown extraction method {method!r}")
ValueError: Unknown extraction method 'enum'

```

**Comment by jpmckinney** [comment]
pip install -e . (mentioned earlier under setup)

File: app/utils/tables.py [hunk 1]
```diff
@@ -10,10 +10,10 @@
 
 
 def _format_currency(number: Decimal | None, currency: str) -> str:
-    if isinstance(number, str):
+    if isinstance(number, str) or not number:
```
Comment (jpmckinney) [inline app/utils/tables.py:13]: Why not 

```
if number is None:
    return "-"
```

Seems odd to call int on values like None, “” and 0.

Comment (yolile) [reply app/utils/tables.py:13]: I did that first but wasn't sure what was better, an additional if/return or the current approach. I will change it to the additional if then

Comment (jpmckinney) [reply app/utils/tables.py:13]: Clarity over brevity :)

**Comment by yolile** [comment]
I'm not sure why "Not authenticated" is not being translated, not sure if we are mocking JWT somewhere

**Comment by jpmckinney** [comment]
We can create a follow-up issue to translate model names, if that's desired.

**Pull request merged**